### DISCUSSION TITLE
Problem with UPFD dataset

### DISCUSSION THREAD

**Discussion by AmazingShadow** [pyg-team/pytorch_geometric#8760] [Q&A]
I had a problem with UPFD dataset, this is my code:
train_data = UPFD(root="./content", name="politifact", feature="content", split="train")

Problem is "BadZipFile: File is not a zip file" when I try run my code

**Comment by rusty1s** [comment]
Thanks for reporting. Will be fixed in https://github.com/pyg-team/pytorch_geometric/pull/8766
